ssapy/compute.py

```diff
@@ -1209,19 +1210,20 @@ def _nby3shape(arr_):
             return arr_.T
 
 
-def keplerian_orbital_elements(r_, v_, mu_barycenter=EARTH_MU):
+def calculate_orbital_elements(r_, v_, mu_barycenter=EARTH_MU):
```

Comment:
Why change the name of this function?

Reply:
I was moving toward a more standardized way of calling these type of functions that calculate something using the ssapy position outputs.